python/oneflow/test/modules/test_global_cast.py

```diff
@@ -0,0 +1,133 @@
+"""
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import unittest
+
+import oneflow as flow
+from oneflow import nn
+import oneflow.unittest
+from oneflow.nn.graph.util import IONodeType, IONode
+
+
+class GraphModel(nn.Graph):
+    def __init__(self, model):
+        super().__init__()
+        self.model = model
+    
+    def build(self, x):
+        return self.model(x)
+
+
+class TestToGlobalLocal(oneflow.unittest.TestCase):
+    placement = flow.placement('cpu', ranks=[0, 1])
+    sbp = (flow.sbp.broadcast,)
+    model = nn.Sequential(nn.Linear(8, 4), nn.ReLU(), nn.Linear(4, 2))
+    local_graph_model = GraphModel(model)
+    global_graph_model = GraphModel(model.to_global(placement=placement, sbp=sbp))
+
+    def _all_global(test_case, input, placement, sbp):
+        node_tree = IONode(value=input)
+        for _, node in node_tree.named_nodes():
+            if node._type == IONodeType.TENSOR:
+                value = node._value
+                test_case.assertTrue(value.is_global)
+                # check placement
+                test_case.assertEqual(placement.type, value.placement.type)
+                test_case.assertListEqual(list(placement.ranks), list(value.placement.ranks))
+                # check sbp
+                test_case.assertTupleEqual(sbp, value.sbp)
+
+    def _all_local(test_case, input):
+        node_tree = IONode(value=input)
+        for _, node in node_tree.named_nodes():
+            if node._type == IONodeType.TENSOR:
+                test_case.assertFalse(node._value.is_global)
+
+    def test_any_input(test_case):
+        tensor = flow.zeros((3, 4))
+        tensor_lt = [flow.tensor([1, 2, 3]), flow.randn((2, 3, 4))]
+        tensor_tp = (flow.zeros((2, 2)), flow.ones((2, 3)), flow.randn((3, 5)))
+        tensor_dt = {'tensor': tensor, 'tensor_lt': tensor_lt}
```

Comment:
```suggestion
        tensor_list = [flow.tensor([1, 2, 3]), flow.randn((2, 3, 4))]
        tensor_tuple = (flow.zeros((2, 2)), flow.ones((2, 3)), flow.randn((3, 5)))
        tensor_dict = {'tensor': tensor, 'tensor_lt': tensor_lt}
```

这里的用全称吧，用缩写但是又不是很常见，阅读的任反应不过来。其它相应位置都改一下。

Reply:
已修改